packages/manager/apps/pci-savings-plan/src/hooks/useSavingsPlan.tsx

```diff
@@ -182,11 +192,21 @@ export const useSavingsPlanCreate = (
       displayName,
       offerId,
       size,
-    }: {
-      displayName: string;
-      offerId: string;
-      size: number;
-    }) => postSavingsPlan({ serviceId, offerId, displayName, size }),
+      startDate,
+    }: MutationCreatePlanParams) => {
+      const year = startDate.getFullYear();
+      const month = String(startDate.getMonth() + 1).padStart(2, '0');
+      const day = String(startDate.getDate()).padStart(2, '0');
+      const date = `${year}-${month}-${day}`;
```

Comment:
I think this type of date formatter would have been well suited to a utils, as it's a classic format that could be used again, and also could have been accompanied by tests to ensure its robustness.